toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -409,6 +420,59 @@ os:
       permissions: "664"
 ```
 
+## dirConfig type
+
+Specifies options for placing a directory in the OS.
+
+Type is used by: [additionalDirs](#additionaldirs-dirconfig)
+
+<div id="dirconfig-path"></div>
+
+### sourcePath [string]
+
+The absolute path to the source directory that will be copied.
+
+### destinationPath [string]
+
+The absolute path in the target OS that the source directory will be copied to.
+
+Example:
+
+```yaml
+os:
+  additionalDirs:
+    - sourcePath: "home/files/targetDir"
+      destinationPath: "usr/project/targetDir"
+```
+
+### newDirPermissions [string]
+
+The permissions to set on all of the new directories being created on the target OS (including the top-level directory). Default value: `755`.
+
+### mergedDirPermissions [string]
+
+The permissions to set on the directories being copied that already do exist on the target OS (including the top-level directory). **Note:** If this value is not specified in the config, the permissions for this field will be the same as that of the pre-existing directory.
+
+### childFilePermissions [string]
+
+The permissions to set on the children file of the directory. Default value: `755`.
+
+Supported formats for permission values:
+
+- Int containing an octal value. e.g. `664`
```

Comment:
An integer can't contain an octal value. Octal is a string representation of a number.